zerver/lib/push_notifications.py

```diff
@@ -547,6 +547,9 @@ def add_push_device_token(
         post_data = {
             "server_uuid": settings.ZULIP_ORG_ID,
             "user_uuid": str(user_profile.uuid),
+            # user_id is sent so that the bouncer can delete any pre-existing registrations
+            # for this user+device to avoid duplication upon adding the uuid registration.
+            "user_id": str(user_profile.id),
```

Comment:
The fact that we need this means we need to backport this commit, right?

Reply:
Yeah, probably - that will let us stop potentially creating duplicates sooner, and there's probably no reason to wait with this for 7.0?

Reply:
OK, tagging as a backport candidate.